apps/client/src/app/components/rule/rule-settings-dialog/rule-settings-dialog.html

```diff
@@ -1,76 +1,126 @@
-    </mat-slider>
-    @if (data.rule.configuration.threshold.unit === '%') {
-      <label>{{
-        data.rule.configuration.threshold.max | percent: '1.2-2'
-      }}</label>
-    } @else {
-      <label>{{ data.rule.configuration.threshold.max }}</label>
-    }
-  </div>
-  <div
-    class="w-100"
-    [ngClass]="{ 'd-none': !data.rule.configuration.thresholdMax }"
-  >
-    <h6 class="mb-0">
-      <ng-container i18n>Threshold Max</ng-container>:
+      <h6 class="mb-0">
+        <ng-container i18n>Threshold Min</ng-container>:
+        @if (data.rule.configuration.threshold.unit === '%') {
+          {{ data.settings.thresholdMin | percent: '1.2-2' }}
+        } @else {
+          {{ data.settings.thresholdMin }}
+        }
+      </h6>
       @if (data.rule.configuration.threshold.unit === '%') {
-        {{ data.settings.thresholdMax | percent: '1.2-2' }}
+        <label>{{
+          data.rule.configuration.threshold.min | percent: '1.2-2'
+        }}</label>
       } @else {
-        {{ data.settings.thresholdMax }}
+        <label>{{ data.rule.configuration.threshold.min }}</label>
       }
-    </h6>
-    @if (data.rule.configuration.threshold.unit === '%') {
-      <label>{{
-        data.rule.configuration.threshold.min | percent: '1.2-2'
-      }}</label>
-    } @else {
-      <label>{{ data.rule.configuration.threshold.min }}</label>
-    }
-    <mat-slider
-      name="thresholdMax"
-      [max]="data.rule.configuration.threshold.max"
-      [min]="data.rule.configuration.threshold.min"
-      [step]="data.rule.configuration.threshold.step"
+      <mat-slider
+        name="thresholdMin"
+        [max]="data.rule.configuration.threshold.max"
+        [min]="data.rule.configuration.threshold.min"
+        [step]="data.rule.configuration.threshold.step"
+      >
+        <input matSliderThumb [(ngModel)]="data.settings.thresholdMin" />
+      </mat-slider>
+      @if (data.rule.configuration.threshold.unit === '%') {
+        <label>{{
+          data.rule.configuration.threshold.max | percent: '1.2-2'
+        }}</label>
+      } @else {
+        <label>{{ data.rule.configuration.threshold.max }}</label>
+      }
+    </div>
+    <div
+      class="w-100"
+      [ngClass]="{ 'd-none': !data.rule.configuration.thresholdMax }"
     >
-      <input matSliderThumb [(ngModel)]="data.settings.thresholdMax" />
-    </mat-slider>
-    @if (data.rule.configuration.threshold.unit === '%') {
-      <label>{{
-        data.rule.configuration.threshold.max | percent: '1.2-2'
-      }}</label>
-    } @else {
-      <label>{{ data.rule.configuration.threshold.max }}</label>
-    }
-  </div>
+      <h6 class="mb-0">
+        <ng-container i18n>Threshold Max</ng-container>:
+        @if (data.rule.configuration.threshold.unit === '%') {
+          {{ data.settings.thresholdMax | percent: '1.2-2' }}
+        } @else {
+          {{ data.settings.thresholdMax }}
+        }
+      </h6>
+      @if (data.rule.configuration.threshold.unit === '%') {
```

Comment:
Can you wrap the slider including labels with `<div class="align-items-center d-flex w-100">`